radio/src/lua_api.cpp

```diff
@@ -129,6 +129,29 @@ static int luaGetDateTime(lua_State *L)
   return 1;
 }
 
+static void luaPushLatLng(TelemetrySensor & telemetrySensor, TelemetryItem & telemetryItem)
+/* result is lua table containing members ["Lat"] and ["Lng"] as lua_Number (doubles) in decimal degrees */
+{
```

Comment:
We use `Lon` abbreviation for Longitute, please rename it everywhere you used `Lng`.